ivadomed/loader/loader.py

```diff
@@ -736,9 +736,9 @@ def __getitem__(self, index):
         # Clean transforms params from previous transforms
         # i.e. remove params from previous iterations so that the coming transforms are different
         # Use copy to have different coordinates for reconstruction for a given handler with patch
-        metadata_input = imed_loader_utils.clean_metadata(copy.deepcopy(seg_pair_slice['input_metadata']))
-        metadata_roi = imed_loader_utils.clean_metadata(copy.deepcopy(roi_pair_slice['gt_metadata']))
-        metadata_gt = imed_loader_utils.clean_metadata(copy.deepcopy(seg_pair_slice['gt_metadata']))
```

Comment:
I had trouble understanding what `clean_metadata` was supposed to do because with the `copy.deepcopy`, it seems that "old" transforms from a previous epoch are not there anyway. My understanding is that if we keep `deepcopy` (here or at line 724) `clean_metadata` is not necessary, but this would need to be confirm.

As a reference, I was the one who add the `copy.deepcopy` in 2D in PR #763  to be able to keep the patch coordinates (same as what was already in 3D for sub-volumes).

Reply:
I agree with you @mariehbourget : I think we can get ride of `clean_metadata`!